### ISSUE TITLE
A2: Creating a smart contract

### ISSUE THREAD

**Issue by pancaldi** [IBM-Blockchain/blockchain-vscode-extension#2405]
## Description
<!I was creating a smart contract (demo-contract) and at the end I got the message that it was successfully created. Then on to step A3.2, Fabric Environments did not display '1 Org Local Fabric o (click to start)', it only displayed 'click + to add environments'. How to solve this issue? --- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug. What were you hoping to achieve? -->

## Expected Behavior
<Fabric Environments should display '1 Org Local Fabric o (click to start)' !--- Tell us what should happen -->

## Actual Behavior
< click + to add environments !--- Tell us what happens instead -->

## Steps to Reproduce

1.
2.
3.

## Existing issues

- [ ] [Stack Overflow issues](https://stackoverflow.com/questions/tagged/ibp-vscode-extension)
- [ ] [GitHub Issues](https://github.com/IBM-Blockchain/blockchain-vscode-extension/issues)

## Your Environment

* IBM Blockchain Platform Extension Version:
* Visual Studio Code version:
* Environment name and version (e.g. Node.js v8.12.0, npm v6.4.1):
* Operating System and version:
* Link to your project:

**Comment by Jakeeyturner** [comment]
Hi @pancaldi,

The `1 Org Local Fabric` might not be in the Fabric Environments panel as at some point you might've deleted it (and it won't recreate itself).

Thanks,
Jake

**Comment by pancaldi** [comment]
But how do I fix this issue?Sent from my T-Mobile 4G LTE Device
-------- Original message --------From: Jake <notifications@github.com> Date: 6/1/20  04:16  (GMT-05:00) To: IBM-Blockchain/blockchain-vscode-extension <blockchain-vscode-extension@noreply.github.com> Cc: Jacques Bikoundou <jaqo59@gmail.com>, Mention <mention@noreply.github.com> Subject: Re: [IBM-Blockchain/blockchain-vscode-extension] A2: Creating a smart contract (#2405) 
Hi @pancaldi,
The 1 Org Local Fabric might not be in the Fabric Environments panel as at some point you might've deleted it (and it won't recreate itself).
Thanks,
Jake

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/IBM-Blockchain/blockchain-vscode-extension/issues/2405#issuecomment-636688139",
"url": "https://github.com/IBM-Blockchain/blockchain-vscode-extension/issues/2405#issuecomment-636688139",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by Jakeeyturner** [comment]
- Click on 'Add new environment'
- Select 'Create new from template'
- Select '1 Org template'
- Provide a name for the environment such as `1 Org Local Fabric`

**Comment by pancaldi** [comment]
Thanks buddy!Right now I am stuck at steps A39, A40 of the tutorial. I created a query.ts file off create.ts and I needed to replace the lines that 'Submit the transaction ' but I am unable to edit or copy/paste. I can remove the line but I can't paste or type in the 'Evaluate the specified transaction'. I checked all my settings and nothing is responding! Thanks Sent from my T-Mobile 4G LTE Device
-------- Original message --------From: Jake <notifications@github.com> Date: 6/1/20  09:16  (GMT-05:00) To: IBM-Blockchain/blockchain-vscode-extension <blockchain-vscode-extension@noreply.github.com> Cc: Jacques Bikoundou <jaqo59@gmail.com>, Mention <mention@noreply.github.com> Subject: Re: [IBM-Blockchain/blockchain-vscode-extension] A2: Creating a smart contract (#2405) 

Click on 'Add new environment'
Select 'Create new from template'
Select '1 Org template'
Provide a name for the environment such as 1 Org Local Fabric


—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/IBM-Blockchain/blockchain-vscode-extension/issues/2405#issuecomment-636855090",
"url": "https://github.com/IBM-Blockchain/blockchain-vscode-extension/issues/2405#issuecomment-636855090",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
